search.py

```diff
@@ -0,0 +1,77 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+import os
+import json
+import sys
+import dotenv
+import louis.models.openai as openai
+import louis.db as db
+from microbench import MicroBench
+import pandas as pd
+
+
+# This is used to load the .env file
+dotenv.load_dotenv()
+basic_bench = MicroBench()
+WEIGHTS = os.environ.get("WEIGHTS")
```

Comment:
FINESSE_WEIGHTS

Environment variables should be prefixed with the products they apply to.

also as discussed, we should support both filename and string. perhaps instead of JSON when setting it up as a string, we can have this format:

FINESSE_WEIGHTS=traffic:1, current:0.5, typicality:0.2